src/integration_tests.zig

```diff
@@ -150,7 +150,7 @@ test "benchmark smoke" {
 
     const tigerbeetle = try tigerbeetle_exe(shell);
     const status_ok = try shell.exec_status_ok(
-        "{tigerbeetle} benchmark --transfer-count=4000",
+        "{tigerbeetle} benchmark --transfer-count=10_000 --transfer-batch-size=10 --validate",
```

Comment:
Bumped the `transfer-count` up and the `transfer-batch-size` down to squeeze a bit more surface area from this run, while still being fast.